### PR TITLE
Copy-poll work

### PR DESCRIPTION
**Pull request by JDawes-ScottLogic**

### CODE REVIEW

**Comment by stevenhillcox** [comment]
This looks good to me.

**Comment by JDawes-ScottLogic** [comment]
Found an issue with options not copying. Currently investigating.

File: VotingApplication/VotingApplication.Web/Api/Controllers/DashboardController.cs [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+            }
+        }
+
+        private static Poll CopyPoll(Poll pollToCopy)
+        {
+            return new Poll()
+            {
+                UUID = Guid.NewGuid(),
+                ManageId = Guid.NewGuid(),
+                Name = "Copy of " + pollToCopy.Name,
```
Comment (tpkelly) [inline VotingApplication/VotingApplication.Web/Api/Controllers/DashboardController.cs]: Not sure we should have the "Copy of", as the poll name isn't unique. If i'm copying "Friday Lunch", I would expect another "Friday Lunch"

Comment (JDawes-ScottLogic) [inline VotingApplication/VotingApplication.Web/Api/Controllers/DashboardController.cs]: Currently it's not particularly easy to distinguish different polls on the dashboard. The copy would be at the top of the list, but that's the only way to tell them apart.

**Pull request merged**